boards/arm64/fvp-v8r/fvp-armv8r/configs/nsh/defconfig

```diff
@@ -5,74 +5,46 @@
 # You can then do "make savedefconfig" to generate a new defconfig file that includes your
 # modifications.
 #
-CONFIG_ARCH="arm64"
-CONFIG_ARCH_ARM64=y
-CONFIG_ARCH_BOARD="fvp-armv8r"
-CONFIG_ARCH_BOARD_FVP_ARMV8R=y
-CONFIG_ARCH_CHIP="fvp-v8r"
-CONFIG_ARCH_CHIP_FVP_ARMV8R=y
-CONFIG_ARCH_CHIP_FVP_R82=y
-CONFIG_ARCH_EARLY_PRINT=y
-CONFIG_ARCH_INTERRUPTSTACK=4096
-CONFIG_ARM64_STRING_FUNCTION=y
+# CONFIG_ARCH_FPU is not set
+# CONFIG_NSH_ARGCAT is not set
+# CONFIG_NSH_CMDOPT_HEXDUMP is not set
+CONFIG_ARCH="arm"
+CONFIG_ARCH_BOARD="stm32f4discovery"
```

Comment:
this doesn't look good :)

Reply:
wow! Good finding!

I just run ./tools/refresh.sh and it created these modifications!